### PR TITLE
[tests-only][full-ci]Extended public link tests to have more checks

### PR DESCRIPTION
**Pull request by SwikritiT**
## Related Issue

- Part of: https://github.com/owncloud/ocis/issues/4937

## How Has This Been Tested?

- locally
- CI

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Technical debt
- [x] Tests only (no source changes)

## Checklist:

- [ ] Code changes
- [ ] Unit tests added
- [x] Acceptance tests added
- [ ] Documentation ticket raised: <link>

### CODE REVIEW

File: tests/acceptance/features/apiSpacesShares/shareSubItemOfSpaceViaPublicLink.feature [hunk 1]
```diff
@@ -44,17 +44,24 @@ Feature: Share a file or folder that is inside a space via public link
     And the OCS status code should be "200"
     And the OCS status message should be "OK"
     And the fields of the last response to user "Alice" should include
-      | path | <path> |
+      | item_type         | <item_type>          |
+      | mimetype          | <mimetype> |
+      | file_target       | <file_target>        |
```
Comment (saw-jan) [inline tests/acceptance/features/apiSpacesShares/shareSubItemOfSpaceViaPublicLink.feature:49]:
```suggestion
      | item_type         | <item_type>          |
      | mimetype          | <mimetype>           |
      | file_target       | <file_target>        |
```

File: tests/acceptance/features/apiSpacesShares/shareSubItemOfSpaceViaPublicLink.feature [hunk 2]
```diff
@@ -70,11 +77,17 @@ Feature: Share a file or folder that is inside a space via public link
     And the OCS status code should be "200"
     And the OCS status message should be "OK"
     And the fields of the last response to user "Alice" should include
-      | path | <path> |
+      | item_type         | <item_type>   |
+      | mimetype          | <mimetype>    |
+      | file_target       | <file_target> |
+      | path              | /<entity>      |
+      | share_type        | public_link   |
```
Comment (saw-jan) [inline tests/acceptance/features/apiSpacesShares/shareSubItemOfSpaceViaPublicLink.feature:84]:
```suggestion
      | file_target       | <file_target> |
      | path              | /<entity>     |
      | share_type        | public_link   |
```

**Review by SagarGi — COMMENTED** [review]
lgtm :+1:

**Review by saw-jan — COMMENTED** [review]
LGTM :+1:

**Review by grgprarup — COMMENTED** [review]
:+1:

**Review by phil-davis — APPROVED** [review]

**Review by ScharfViktor — APPROVED** [review]
LGTM

**Comment by phil-davis** [comment]
Ported to master in PR #5335

**Pull request merged**